src/components/ha-input-helper-text.ts

```diff
@@ -0,0 +1,25 @@
+import { css, html, LitElement, TemplateResult } from "lit";
+import { customElement } from "lit/decorators";
+
+@customElement("ha-input-helper-text")
+class InputHelperText extends LitElement {
+  protected render(): TemplateResult {
+    return html`<slot></slot>`;
+  }
+
+  static styles = css`
+    :host {
+      display: block;
+      color: var(--mdc-text-field-label-ink-color, rgba(0, 0, 0, 0.6));
```

Comment:
```suggestion
      color: var(--secondary-text-color, rgba(0, 0, 0, 0.6));
```

Should we use this?

Reply:
Right now I have taken the exact variables as used by the `ha-textfield` component.